test/__snapshots__/threeIdProvider.test.js.snap

```diff
@@ -1,173 +1,173 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ThreeIdProvider Unsupported method should throw 1`] = `[Error: Unsupported method: 3id_notSupported]`;
+exports[`ThreeIdProvider Unsupported method should throw 1`] = `[Error: Method not found]`;
 
-exports[`ThreeIdProvider Unsupported method should throw 2`] = `"Unsupported method: 3id_notSupported"`;
+exports[`ThreeIdProvider Unsupported method should throw 2`] = `[Error: Method not found]`;
 
 exports[`ThreeIdProvider asymmetrically decrypt correctly 1`] = `
 Object {
   "id": 1,
-  "json-rpc": "2.0",
+  "jsonrpc": "2.0",
```

Comment:
Nice catch. Didn't realize we where doing this wrong!

Reply:
I got the issue running the tests, hopefully using https://github.com/ceramicnetwork/js-rpc-utils should help avoid these kind of issues!